Summit/2025/program.html

```diff
@@ -21,13 +21,419 @@
   heading = "Program-at-a-glance"
 %}
 
-
 <div class="w-full md:w-3/4 lg:w-3/4 mt-8 mx-auto aspect-5/2">
-  {% svg /images/summit/program-early-2025.drawio.svg width="100%" %}
+  {% svg /images/summit/program-at-a-glance-2025.drawio.svg width="100%" %}
 </div>
 
-<h3 class="pt-20 pb-20 text-center h3-size text-dark">
-  Detailed program coming soon. Stay tuned!
-</h3>
+{% include section-end-without-up-arrow.html %}
+
+
+
+<!-- Main Program section -->
+{% include section-start.html
+  id = "program"
+  heading = "Main Program"
+  intro = '
+    <p>
+      All times are local time in Prague, Czech Republic (GMT+2). <!--Session chairs shown on right.-->
```

Comment:
If you still have some talks that are not 100% confirmed (eg when presenter said they aren't sure they can come), then maybe say "subject to small adjustments" or something similar